prismic-model/json/pages.json

```diff
@@ -279,9 +279,9 @@
               }
             }
           },
-          "venueTimes": {
+          "collectionVenue": {
             "type": "Slice",
-            "fieldset": "Venue opening times",
+            "fieldset": "Collection venue",
```

Comment:
It doesn't look like the JSON has been updated?
And are we sure these were not used anywhere?

Reply:
- Do you mean the json for all the other content types that share Body?
- Yeah, this slice was created for this piece of work, so nothing else uses it

Reply:
The point of the body though was to be generic, [see here](https://github.com/wellcometrust/wellcomecollection.org/issues/4294).

By creating forks, it feels like we would have to come back to this all the time, making it an unscalable bit of work stopping us from getting onto more interesting things.

Reply:
Think we might be talking at cross purposes, I'll come and chat